CONTRIBUTING.adoc

```diff
@@ -634,9 +634,9 @@ Sample of such metadata file:
 ```yaml
 ---
 name: 'My Jenkins'
-url: '/images/logos/${imageId}/${imageId}.png'
-url_256: '/images/logos/${imageId}/256.png'
-vector: '/images/logos/${imageId}/${imageId}.svg'
+url: '/logos/${imageId}/${imageId}.png'
+url_256: '/logos/${imageId}/256.png'
+vector: '/logos/${imageId}/${imageId}.svg'
```

Comment:
Let's remove the leading slash as well so that the example matches all the existing usages.  Thanks very much for detecting that mistake and correcting it in the contributing guide.
```suggestion
url: 'logos/${imageId}/${imageId}.png'
url_256: 'logos/${imageId}/256.png'
vector: 'logos/${imageId}/${imageId}.svg'
```